python/tk_framework_desktopserver/command.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, p_out, target_queue):
         :param p_out: Pipe to read.
         :param target_queue: Queue that will accumulate the pipe output.
         """
-        Thread.__init__(self)
```

Comment:
What's that? It's off-topic right?

Reply:
I changed the base branch we can have an accurate diff.